.github/workflows/publish_docker_images_cron.yml

```
@@ -5,16 +5,12 @@ on:
     - cron: 30 0 * * *
 
 jobs:
-
   build_and_publish_images:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        image_type: [ latest,
-                      alpine,
-                      postgres_12,
-                      nightly ]
+        image_type: [latest, alpine, postgres_12, nightly]
```

I don't think that you understood what I meant in https://github.com/citusdata/docker/pull/256#discussion_r693115583

```suggestion
        image_type: 
        - latest
        - alpine
        - postgres_12
        - nightly
```

---

I forgot to change here. I changed **_on_push.yml pipeline